### PR TITLE
Update Part 0: Most conflicts sorted

### PR DESCRIPTION
**Pull request by Jalleo**
Suffer not the outdated code.

### CODE REVIEW

**Comment by Jalleo** [comment]
😱

File: .github/workflows/comp_changelog.yml [hunk 1]
```diff
@@ -2,35 +2,38 @@ name: Compile changelogs
```
Comment (francinum) [inline .github/workflows/comp_changelog.yml:2]: Drop all the chances to this file.

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Docker Image CI
```
Comment (francinum) [inline .github/workflows/docker.yml:1]: @Crossedfall Is any of this relevant to us?

File: SQL/beestation_schema.sql [hunk 3]
```diff
@@ -70,6 +70,7 @@ CREATE TABLE IF NOT EXISTS `SS13_ban` (
   `unbanned_computerid` varchar(32) DEFAULT NULL,
   `unbanned_round_id` int(11) unsigned DEFAULT NULL,
   `global_ban` tinyint(1) unsigned NOT NULL DEFAULT '1',
+  `hidden` tinyint(1) unsigned NOT NULL DEFAULT '0',
```
Comment (francinum) [inline SQL/beestation_schema.sql:73]: DB Schema update. Requires host final approval.

**Comment by Kmc2000** [comment]
Good luck kiddo

File: .github/workflows/comp_changelog.yml (hunk 1 above)
Comment (Jalleo) [reply .github/workflows/comp_changelog.yml:2]: Okay.

File: Dockerfile [hunk 4]
```diff
@@ -1,6 +1,6 @@
-FROM beestation/byond:513.1520 as base
+FROM beestation/byond:513.1521 as base
```
Comment (Jalleo) [inline Dockerfile:1]: seemed like a good idea to update these two as bee had it upgraded.

File: .github/workflows/comp_changelog.yml (hunk 1 above)
Comment (Jalleo) [reply .github/workflows/comp_changelog.yml:2]: Done

**Pull request merged**